scripts/target_determinator.py

```diff
@@ -59,6 +60,10 @@ def filter_targets(bazel: Path, targets: str) -> str:
             return p.stdout
         except subprocess.CalledProcessError as err:
             log(err.stderr)
+            log("Query file:\n```")
+            with open(tmp.name) as f:
+                log(f.read())
+            log("```")
```

Comment:
```suggestion
            log("Full query file:\n```\n")
            with open(tmp.name) as f:
                log(f.read())
            log("```")
```

Reply:
Log uses `print` which I thought already adds a newline at the end?

Reply:
I tweaked the log string, but the docs for `print` indicate it includes a newline at the end.